code/modules/organs/ailments/_ailment.dm

```diff
@@ -0,0 +1,96 @@
+/datum/ailment
+	var/name
+	var/timer_id
+	var/min_time = 2 MINUTES
+	var/max_time = 5 MINUTES
+	var/category = /datum/ailment
+	var/obj/item/organ/organ
+
+	// Requirements before applying to a target.
+	var/list/applies_to_organ
+	var/specific_organ_subtype = /obj/item/organ/external
+	var/affects_robotics = FALSE
+
+	var/treated_by_item_type
```

Comment:
Add comment about what this var takes (type? list of types?), as this is probably what anyone touching this system will want to touch.